controllers/controllers.js

```diff
@@ -55,4 +55,18 @@ function sendAllComments(req, res, next){
     })
 }
 
-module.exports = { sendTopics, sendAllEndpoints, sendArticleById, sendAllArticles, sendAllComments }
\ No newline at end of file
+function postComments(req, res, next){
+    // console.log(req.body)
+    const {article_id} = req.params;
+    const {username, body} = req.body;
+    // console.log(username)
+    return addComments(username, body, article_id)
+    .then((comment) => {
+        res.status(200).send({comment})
+    })
+    .catch((err) => {
+        next(err)
+    })
```

Comment:
Don't forget to remove console logs from your code. Keep it looking nice and tidy!